crystalfontz/dbus/domain/keys.py

```diff
@@ -0,0 +1,63 @@
+from typing import ClassVar, Tuple
+
+from crystalfontz.dbus.domain.base import ByteM, struct
+from crystalfontz.keys import KeyState, KeyStates
+
+KeyPressT = int
+
+
+class KeyPressM(ByteM):
+    t: ClassVar[str] = ByteM.t
+
+
+KeyStateT = Tuple[bool, bool, bool]
+KeyStatesT = Tuple[
+    KeyStateT,
+    KeyStateT,
+    KeyStateT,
+    KeyStateT,
+    KeyStateT,
+    KeyStateT,
+]
+
+
+class KeyStateM:
```

Comment:
Do any of these need docs? I'm trying to only document the "public" domain types.

Reply:
None of them are exposed publicly, as far as I can tell. But this will be easy to double-check when generating dbus documentation.